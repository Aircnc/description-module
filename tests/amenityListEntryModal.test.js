import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import AmenityListEntryModal from '../client/src/components/amenityListEntryModal.jsx';

describe('AmenityListEntryModal', () => {
  it('should render correctly', () => {
    var output = shallow(
      <AmenityListEntryModal amenity='mockAmenity' isLast='mockIsLast' notAvailable='mockNotAvailable'/>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});