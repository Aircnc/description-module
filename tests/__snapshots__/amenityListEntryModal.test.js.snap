// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AmenityListEntryModal should render correctly 1`] = `
<div>
  <styled.div
    notAvailable="mockNotAvailable"
  >
    mockAmenity
  </styled.div>
  <styled.div
    notAvailable="mockNotAvailable"
  >
    A description for this item is not available at this time
  </styled.div>
  <styled.div
    isLast="mockIsLast"
  />
</div>
`;
